tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/types/ImportUserDefinedTypeTest.java

```diff
@@ -0,0 +1,34 @@
+package org.ballerinalang.test.bala.types;
+
+import org.ballerinalang.test.BCompileUtil;
+import org.ballerinalang.test.CompileResult;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+/**
+ * Test cases for import user defined type.
+ */
+public class ImportUserDefinedTypeTest {
+    private CompileResult result;
```

Comment:
Given that `result` is used in only one test method we can make it a private variable there and get rid of the `tearDown` method,